app/controllers/api/v1/notes_controller.rb

```diff
@@ -0,0 +1,74 @@
+module Api
+  module V1
+    class NotesController < ApplicationController
+      def index
+        return render_long_page_size if page_size.to_i > max_page_size
+        return render_invalid_type if type.present? && invalid_note_type
```

Comment:
esoo, a esto queríamos llegar!

Si queremos ir un pasito mas (no hace falta ahora)
cada condición del if podría ser un método también, no?

Por ejemplo:

```suggestion
        return render_long_page_size if invalid_page_size?
        return render_invalid_type if invalid_type?
```

```ruby
def invalid_page_size?
  page_size.to_i > max_page_size
end

def invalid_type?
  type.present? && invalid_note_type
end
```

Reply:
Listoo